### ISSUE TITLE
Answer is always rendered "Yes"

### ISSUE THREAD

**Issue by DataGraham** [enquirer/prompt-confirm#11]
Code to reproduce:
```
new Confirm('Proceed?').run()
```

Output:
```
? Proceed? (Y/n) n
? Proceed? Yes
```

The promise still resolves to false; only the rendered output is incorrect.

**Comment by doowb** [comment]
Thanks, I think I know where the bug is and will publish a patch soon.

**Comment by doowb** [comment]
Published in `2.0.1`.

**Comment by DataGraham** [comment]
Yep, fixed. Thanks!
